resource/build2019-prod/session_length_map.json

```diff
@@ -0,0 +1,43 @@
+{
+    "SessionLengthMap": [{
+            "SessionTypeName": "Breakout",
```

Comment:
Can we please have each of the session types from the session set map listed in here?